README.md

```diff
@@ -20,40 +20,48 @@ npm run build
 npm run lint
 ```
 
-## For the mobile app
+### Customize configuration
+See [Configuration Reference](https://cli.vuejs.org/config/).
+
+# Mobile app
 
 In order to be able to build the mobile app, you will need to follow these instructions first:
 [capacitor environment setup instructions](https://capacitorjs.com/docs/getting-started/environment-setup)
 
-### To generate the splash screens and the icons for mobile deployment
+## To generate the splash screens and the icons for mobile deployment
 
 If you need to do this, make sure you have cordova-res installed with `npm i -g cordova-res`
 
 Then put your icon and splash files in `resources/icon.png` and `resources/splash.png` respectively (see the files for minimum sizes) and then run:
 
 ```
+cd project root
 cordova-res android --skip-config --copy && cordova-res ios --skip-config --copy
 ```
 
-### To build the assets (the web app) for mobile deployment
+## Theme customization
+You can customize the app's look by setting your values in the `<project root>/public/config.json` file or by putting a separate scss variables declarations file in `<project root>/public/config.theme.scss` (see the `<project root>/src/assets/custom-variables.scss` for a reference of what you can customize). Note that if you set your variables in the `config.json` file, you **must** remove the `$` prepending the sass variables before setting them as json keys.
+
+## To build the assets (the web app) for mobile deployment
+**!IMPORTANT** : Do not forget to put your config file at `<project root>/public/config.json` otherwise the build will exit with an error
 
-*!IMPORTANT* : Do not forget to put your config file at `<project root>/public/config.json` otherwise the build will exit with an error
+
+## Prepare app for build
 
 ```
 npm run build:mobile // This will build the assets with the content of the aforementioned config file
 npx cap sync // this will copy the assets into the android and ios directories and make sure the platforms have all their needed dependencies
 ```
 
-### To run in ios or android simulator
+## Run in simulator
```

Comment:
note that this section is also the way to run it on a real android device. This might be cleared out. If you didn't know, you just need to plug your android phone (configured with developper mode) and the `npx cap run android` command will ask you if you want to run it on you plugged in device.

Reply:
Did not see the highlight. Might be time to clean my glasses.